.github/workflows/deploy-artifacts/entrypoint.sh

```diff
@@ -345,10 +345,15 @@ main() {
   structure_build_artifacts
   cd structured_build_artifacts
   # Upload all packages
+  cd rpm
   upload_rpm_packages
+  cd ..
+  cd deb
```

Comment:
Why did this previously work for debs uploaded to JFrog but not for rpms?
was that just JFrog behavior?